### ISSUE TITLE
Regression with SQL errors related to tornado->requests switch

### ISSUE THREAD

**Issue by torsstei** [IBM-Cloud/sql-query-clients#45]
In certain cases we run into this error when a SQL syntax error is occurring:

```---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-36-85cd646704b3> in <module>()
     55 """
     56 
---> 57 run_sql(sql)

<ipython-input-25-73d30e7f4b76> in run_sql(sql_text)
      2     sqlClient.logon()
      3     try:
----> 4         jobId = sqlClient.submit_sql(sql_text)
      5     except SyntaxError as e:
      6         return "SQL job submission failed. {}".format(str(e)), None, None

/opt/conda/envs/DSX-Python35/lib/python3.5/site-packages/ibmcloudsql/SQLQuery.py in submit_sql(self, sql_text, pagesize)
    133                 json=sqlData)
    134             resp = response.json()
--> 135             return resp['job_id']
    136         except HTTPError as e:
    137             raise SyntaxError("SQL submission failed: {}".format(response.json()['errors'][0]['message']))

KeyError: 'job_id'```

Saw it in end user scenario. Trying to find a good repro...

**Comment by gmoehler** [comment]
I can simply reproduce this with a sql not containing a FROM clause:
```
SELECT *
```
Also see this notebook for the complete reproduction: https://dataplatform.cloud.ibm.com/analytics/notebooks/v2/b3fe1127-d4ca-40d5-93cd-ff59c7677aa0/view?access_token=9e89c1c4601da35cc3c7749f36966eb62c8c6a6bbe2bca86fedd42b8385925f7

**Comment by torsstei** [comment]
@emmadickson I just found that there are additional regressions due to the switch to requests lib. Running the standard `ibmcloudsql/test.py` reveals that also list_results() and delete_results() are broken because the GET response body is not returned in the property `body` anymore.

**Comment by emmadickson** [comment]
@torsstei I'm working on this but have some tasks for Insights Squad I have to complete as well. Hoping to put a pr by Wednesday.

**Comment by torsstei** [comment]
Fixed with https://github.com/IBM-Cloud/sql-query-clients/pull/47 and released with v 0.3.3.
